### PR TITLE
Rename pizza models and add API v2

### PR DESCRIPTION
**Pull request by se-bastiaan**
Part of #1428 

### Summary
This PR renames some models in the pizzas package to 'food' and thus may break things in the events package.

It also adds the API v2 for the pizzas package, which I called 'food' instead of 'pizzas'. This because we may decide to change the functionality in the future, and offer other foods. Currently that is not possible due to the way the products work, but the endpoint was already prepared to make this possible by adding the event to the url.

The API also returns a list instead of one single food event, which allows us to run multiple food events at the same time in the future. The only real thing to make this possible is changing the views.

### How to test
Steps to test the changes you made:
1. Go to the API docs on /api/docs and select v2.
2. Check the endpoints and try some things out

### CODE REVIEW

**Comment by se-bastiaan** [comment]
> the actual app hasn't been renamed yet, right?

No that would require much more changes

**Comment by Luca-Castelnuovo** [comment]
deploy review

**Review by Luca-Castelnuovo — COMMENTED** [review]
The code looks fine. Can't test it yet because the deploy is broken.

**Comment by se-bastiaan** [comment]
> 
> 
>     * I am also getting a `CSRF token not set` for the POST method of order.
> 
>     * Do we want the URL's of the pizza app to also change?

* How are you executing the request?
* Pretty much out of scope

File: website/pizzas/api/v2/views.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        IsAuthenticatedOrTokenHasScope,
+        DjangoModelPermissionsOrAnonReadOnly,
+    ]
+    required_scopes = ["food:read"]
+
+
+class FoodEventOrderDetailView(
+    RetrieveAPIView, CreateAPIView, UpdateAPIView, DestroyAPIView
+):
+    """Returns details of an event registration."""
```
Comment (DeD1rk) [inline website/pizzas/api/v2/views.py:73]: You didn't replace this docstring

**Comment by DeD1rk** [comment]
deploy review

**Comment by DeD1rk** [comment]
remove review

**Comment by DeD1rk** [comment]
Sadly the review environment doesn't include the changes from #1664. Maybe we could change it to use the post-merge version when automatic merging is possible. Anyway, @se-bastiaan could you merge #1664 into this? Or do we believe that it works alright just based on the code?

**Comment by JobDoesburg** [comment]
> Sadly the review environment doesn't include the changes from #1664. Maybe we could change it to use the post-merge version when automatic merging is possible. Anyway, @se-bastiaan could you merge #1664 into this? Or do we believe that it works alright just based on the code?

I would do a rebase for sure

**Comment by DeD1rk** [comment]
deploy review

File: website/pizzas/api/v2/urls.py [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+    FoodEventOrderDetailView,
+)
+
+urlpatterns = [
+    path("food/events/", FoodEventListView.as_view(), name="food-events-list"),
+    path(
+        "food/events/<int:pk>/", FoodEventDetailView.as_view(), name="food-event-detail"
+    ),
+    path(
+        "food/events/<int:pk>/order",
```
Comment (se-bastiaan) [inline website/pizzas/api/v2/urls.py:17]:
```suggestion
        "food/events/<int:pk>/order/",
```

**Comment by DeD1rk** [comment]
remove review

**Review by DeD1rk — DISMISSED** [review]
Nice stuff

**Review by DeD1rk — APPROVED** [review]

**Pull request merged**